src/CareTogether.Core/Extensions.cs

```diff
@@ -39,5 +39,18 @@ public static Guid UserId(this ClaimsPrincipal principal)
         public static bool CanAccess(this AuthorizedUser user, Guid organizationId, Guid locationId) =>
             user.Principal.HasClaim(Claims.OrganizationId, organizationId.ToString()) &&
             user.Principal.HasClaim(Claims.LocationId, locationId.ToString());
+
+        private static IDictionary<LogType, string> logTypeMap = new Dictionary<LogType, string>()
+        {
+            { LogType.COMMUNITY_EVENT, "communityEventLog" },
+            { LogType.CONTACT_COMMAND_EXECUTED_EVENT, "contactsEventLog" },
+            { LogType.GOAL_COMMAND_EXECUTED_EVENT, "goalsEventLog" },
+            { LogType.REFERRAL_EVENT, "referralsEventLog" },
+        };
+
+        public static string ConvertToString(this LogType logType)
+        {
+            return logTypeMap[logType];
+        }

```

Comment:
You can remove these mappings and just use [`Enum.ToString()`](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tostring?view=net-5.0) to generate the log type names.

Reply:
I had originally wanted to do that but for some reason thought the log type string had to be those exact camel case strings I had in the converter. I'm not sure why, but agree this is much cleaner.